### PR TITLE
Port remaining node tests

### PR DESCRIPTION
**Pull request by markerikson**
- Ported `node_worker`, `node_stepping`, and `node_object_preview`
- Rewrote the "find visible lines" logic to use actual DOM bounding boxes on the target side

### CODE REVIEW

File: packages/e2e-tests/helpers/console-panel.ts [hunk 1]
```diff
@@ -123,8 +123,9 @@ export function findConsoleMessage(
     ? `[data-test-message-type="${messageType}"]`
     : '[data-test-name="Message"]';
 
+  // Use single quotes because a couple messages have double quoted text displayed
```
Comment (bvaughn) [inline packages/e2e-tests/helpers/console-panel.ts:126]: Wonder if we should throw if you pass a string with double quotes? (Or we could switch outer quote styles, depending?)

File: packages/e2e-tests/helpers/source-panel.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import chalk from "chalk";
 import { openDevToolsTab } from ".";
 import { openPauseInformationPanel } from "./pause-information-panel";
 import { openSource, openSourceExplorerPanel } from "./source-explorer-panel";
-import { clearTextArea, debugPrint, delay, forEach, waitFor } from "./utils";
+import { clearTextArea, debugPrint, delay, forEach, waitFor, mapLocators } from "./utils";
```
Comment (bvaughn) [inline packages/e2e-tests/helpers/source-panel.ts:7]: You don't actually use the new `mapLocators` function

**Review by bvaughn — APPROVED** [review]

File: packages/e2e-tests/helpers/source-panel.ts (hunk 2 above)
Comment (markerikson) [reply packages/e2e-tests/helpers/source-panel.ts:7]: Yeah, I started to import it here, then realized that the logic that uses `Array.from()` in this file is subtly different - it's mapping over a subset of the locators, not all of them.  I can remove the import.

Comment (markerikson) [reply packages/e2e-tests/helpers/source-panel.ts:7]: On the other hand I could use that and slice later. That would work too.

**Pull request merged**